server/plugin/api.go

```diff
@@ -56,6 +56,18 @@ type PRDetails struct {
 	Reviews            []*github.PullRequestReview `json:"reviews"`
 }
 
+type FilteredNotification struct {
+	github.Notification
+	HTMLUrl string `json:"html_url"`
+}
+
+type SidebarContent struct {
+	Reviews     []*github.Issue         `json:"reviews"`
+	PRS         []*github.Issue         `json:"prs"`
```

Comment:
```suggestion
	PRs         []*github.Issue         `json:"prs"`
```
`PR` is an abbreviation but `s` here just indicates plural form